api/controllers/ReferralController.js

```diff
@@ -69,4 +69,47 @@ module.exports = {
         });
       });
   },
+  // Lookup user information using custom url
```

Comment:
Since this is a controller method, we can use the multi-line comment block here. It'll help make the code a little more readable too.